public/data/markers.geojson

```diff
@@ -33,7 +33,6 @@
 { "type": "Feature", "properties": { "sequence": 30, "name": "British American Hotel", "imageAlt": null, "address": "78 Riverside Dr. W.",  "longitude": -83.0411917, "latitude": 42.3198982, "extra": false, "id": "bah", "image": "/photos/bah1.png" }, "geometry": { "type": "Point", "coordinates": [ -83.0411917, 42.3198982 ] } },
 { "type": "Feature", "properties": { "sequence": 31, "name": "1945-46 Windsor Slasher", "imageAlt": null, "address": "Caron Ave. & Riverside Drive W.",  "longitude": -83.0485844, "latitude": 42.317664, "extra": false, "id": "ws", "image": null}, "geometry": { "type": "Point", "coordinates": [ -83.0485844, 42.317664 ] } },
 { "type": "Feature", "properties": { "sequence": 32, "name": "623 Riverside Dr. W.", "imageAlt": null, "address": "623 Riverside Dr. W.",  "longitude": -83.0479701, "latitude": 42.3178514, "extra": false, "id": "623rdw", "image": "/photos/623rdw1.png"}, "geometry": { "type": "Point", "coordinates": [ -83.0479701, 42.3178514 ] } },
-{ "type": "Feature", "properties": { "sequence": 33, "name": "Government Park", "imageAlt": null, "address": "Janette Ave. & Riverside Dr. W.",  "longitude": -83.0474467, "latitude": 42.3180037, "extra": false, "id": "gp", "image": null}, "geometry": { "type": "Point", "coordinates": [ -83.0474467, 42.3180037 ] } },
```

Comment:
Saw that we deleted this feature and it's corresponding json file! Was it just because of a lack of content associated?